wiki/Skinning/Compendium/en.md

```diff
@@ -0,0 +1,261 @@
+[o!s]: /wiki/shared/mode/osu.png 
+[o!t]: /wiki/shared/mode/taiko.png 
+[o!c]: /wiki/shared/mode/catch.png 
+[o!m]: /wiki/shared/mode/mania.png 
+
+# Completed skins compendium
+
+![][o!s] This means the skins contains osu!standard elements
+![][o!c] This means the skins contains osu!catch elements
+![][o!t] This means the skins contains osu!taiko elements
+![][o!m] This means the skins contains osu!mania elements
+
+## A
+
+- [Adventure Time](https://osu.ppy.sh/forum/t/124670) by [Rolling Boy](https://osu.ppy.sh/u/1915456)![][o!s][o!t][o!c]
```

Comment:
you can't do that 

Change `![][o!s][o!t][o!c]` to `![][o!s] ![][[o!t] ![][[o!c]`